onnx/defs/nn/defs.cc

```diff
@@ -126,6 +126,111 @@ namespace onnx {
 } // namespace onnx
 
 namespace onnx {
+    std::function<void(OpSchema&)> LpPoolOpSchemaGenerator(const char* name) {
```

Comment:
the idea of FillUsing and Generator is to share some code between similar ops. Do you mind changing the code above to actually share a piece of common descriptions with MaxPooling?

Reply:
Initially I tried that, but the description between Max and average is a little different.